auslib/web/public/swagger/releases_spec.yml

```diff
@@ -0,0 +1,67 @@
+paths:
+  /api/v1/releases:
+    get:
+      operationId: 'auslib.web.common.releases.get_releases'
+      description: >
+        Returns a JSON Object containing metadata about Releases in Balrog’s database. Due to its size, the actual Release “blob” is never returned from this endpoint.
+        There are a few query arguments that affect its response:
+          -If no arguments are provided, it returns information about all of the Releases;
+          -If "product" is passed, only Releases with the given product name will be returned;
+          -If "name_prefix" is passed, only Releases whose name starts with the given prefix will be returned;
+          -If "names_only" is set to true, the response changes format and provides a list of all of the Release names.
```

Comment:
Is it useful to include these big long description when the parameters have their own? If you're going to include this still, please reformat it - it shows up as a big long paragraph in the UI at the moment.